views/item.ejs

```diff
@@ -1,135 +1,129 @@
 <!DOCTYPE html>
 <html lang='en'>
+
 <head>
   <% include ./partials/head %>
-  <% include ./partials/ace-deps %>
+    <% include ./partials/code-mirror-deps %>
```

Comment:
Some of the indentation changes, like here, seem wrong (don't match other lines).